src/Dapr.Client/ConfigurationItem.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+
+namespace Dapr.Client
+{
+    /// <summary>
+    /// Class that represents an item fetched from the Dapr Configuration API.
+    /// </summary>
+    public class ConfigurationItem
+    {
+        /// <summary>
```

Comment:
Feedback: consider making this immutable after construction, to more clearly communicate intent. Not really a big decision.

Reply:
I agree, done.